code/ATMOSPHERICS/components/unary_devices/tank.dm

```diff
@@ -66,5 +44,6 @@
 /obj/machinery/atmospherics/components/unary/tank/air/New()
 	..()
 	var/datum/gas_mixture/air_contents = AIR1
-	air_contents.oxygen = AIR_CONTENTS * 0.2
-	air_contents.nitrogen = AIR_CONTENTS * 0.8
\ No newline at end of file
+	air_contents.assert_gases("o2", "n2")
+	air_contents.gases["o2"][MOLES] = AIR_CONTENTS * 0.2
```

Comment:
Exceptions are still only runtime errors (as opposed to compile time errors). I ag